src/FqnChecker.php

```diff
@@ -33,6 +33,10 @@ public function enterNode(Node $node) {
             return;
         }
 
+        if (in_array($node->name->toString(), ['true', 'false', 'null'])) {
```

Comment:
This should probably use `\strtolower()` on the name and `, true` as third argument to `in_array`.

Reply:
Good catch! Fixed and force-pushed.